src/components/node-list/styles/_filter.scss

```diff
@@ -3,6 +3,31 @@
 @import '../../../styles/variables';
 @import './variables';
 
+//--- Text label ---//
+
+.pipeline-nodelist__row--unchecked {
+  // Fade row text when unchecked
+  .pipeline-nodelist__row__label--kind-filter {
+    opacity: 0.55;
+  }
+
+  // Brighter row text when unchecked and hovered
+  &:hover {
+    .pipeline-nodelist__row__label--kind-filter {
+      opacity: 0.8;
```

Comment:
I wonder whether we should align this opcity value with a value that was defined in the ruleset ( 0.75)? This way it provides more clarity & ailgnment about the opacity rules throughout node-list-row

Reply:
I assume that it was set to this value for a reason, according to a design from @GabrielComymQB. 
It might be that the text looks different at a certain opacity than an icon at the same opacity. I'm disinclined to mess with it, that seems out of scope.